rxandroid/src/main/java/rx/android/view/OnSubscribeViewClick.java

```diff
@@ -75,7 +76,7 @@ public boolean removeOnClickListener(final View.OnClickListener listener) {
 
         @Override
         public void onClick(final View view) {
-            for (final View.OnClickListener listener : listeners) {
+            for (final View.OnClickListener listener: listeners) {
```

Comment:
Can you revert all of these small whitespace changes, unnecessary usages of `final`?